app/assets/stylesheets/application.scss

```diff
@@ -61,3 +61,32 @@ p {
   background-color: $l-gold;
   margin: 20px;
 }
+
+.house-list ul li{
+  display: inline;
+  padding-right: 10px;
+}
+
+.bill-list ul li{
+  display: inline;
+  padding-right: 10px;
+}
+
+img {
+    width:20%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin, width<br>Colon after property should be followed by one space